src/dmd/dsymbol.h

```diff
@@ -94,20 +94,19 @@ void semantic3(Dsymbol *dsym, Scope* sc);
 const char *mangleExact(FuncDeclaration *fd);
 void mangleToBuffer(Dsymbol *s, OutBuffer* buf);
 
-enum PROTKIND
-{
-    PROTundefined,
-    PROTnone,           // no access
-    PROTprivate,
-    PROTpackage,
-    PROTprotected,
-    PROTpublic,
-    PROTexport,
-};
-
 struct Prot
 {
-    PROTKIND kind;
+    enum Kind
+    {
+        PROTundefined,
```

Comment:
Remove `PROT` prefix.

Reply:
@wilzbach I think this PR is ready to go, with the exception of this change.

Reply:
OK done.